x-pack/plugins/ml/public/application/explorer/anomalies_map.tsx

```diff
@@ -195,7 +192,7 @@ export const AnomaliesMap: FC<Props> = ({ anomalies, jobIds }) => {
       })
     );
 
-    setEMSSuggestions(suggestions.filter((s) => s !== null));
+    setEMSSuggestions(suggestions.filter((s) => s !== null) as MLEMSTermJoinConfig[]);
```

Comment:
We have `isDefined` type guard that should allow you to get rid of the manual type casting 
```suggestion
    setEMSSuggestions(suggestions.filter(isDefined));
```

Reply:
Updated in 1a0d5ad9725dcb63cd403ff2fb1263f197e26982